posthog/api/dashboards/dashboard_tiles.py

```diff
@@ -0,0 +1,165 @@
+from typing import Any, Callable, Dict, List, Optional
+
+import structlog
+from rest_framework import serializers, status, viewsets
+from rest_framework.decorators import action
+from rest_framework.exceptions import PermissionDenied, ValidationError
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from posthog.api.forbid_destroy_model import ForbidDestroyModel
+from posthog.api.routing import StructuredViewSetMixin
+from posthog.models import Dashboard, DashboardTile, Insight, User
+from posthog.models.activity_logging.activity_log import Change, Detail, log_activity
+from posthog.permissions import ProjectMembershipNecessaryPermissions, TeamMemberAccessPermission
+
+logger = structlog.get_logger(__name__)
+
+
+class SimpleDashboardTileSerializer(serializers.Serializer):
```

Comment:
What's a ComplexDashboardTileSerializer?

Reply:
Switched to `BasicBlah` over `SimpleBlah` which mimics a few other places. 

Here, it's not serializing the model, "simply" serializing to the connected foreign IDs